HARK/Calibration/Income/IncomeTools.py

```diff
@@ -111,7 +111,7 @@ def AgeLogPolyToGrowthRates(coefs, age_min, age_max):
     return GrowthFac.tolist(), P0
 
 
-def findPermGroFacs(age_min, age_max, age_ret, AgePolyCoefs, ReplRate):
+def find_perm_GroFacs(age_min, age_max, age_ret, AgePolyCoefs, ReplRate):
```

Comment:
This should be find_PermGroFacs

Reply:
Fixed in a PR to this PR's branch: https://github.com/sbenthall/HARK/pull/1